charts/cinder-csi-plugin/templates/controllerplugin-statefulset.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     {{- include "cinder-csi.controllerplugin.labels" . | nindent 4 }}
 spec:
   replicas: 1
+  serviceName: {{ include "cinder-csi.name" . }}-controllerplugin
```

Comment:
The `imagePullPolicy` below is duplicated, do you mind helping to fix?

Reply:
Done